src/test/java/org/folio/rest/impl/OrdersImplTest.java

```diff
@@ -66,6 +66,9 @@
   private static final String INVALID_OFFSET = "?offset=-1";
   private static final String INVALID_LIMIT = "?limit=-1";
 
+  private static final String PO_LINE_STORAGE_ERROR_ID = "orders_storage_error_id";
```

Comment:
In the PR #32 I also introduced some constants like `ID_DOES_NOT_EXIST`, `ID_FOR_INTERNAL_SERVER_ERROR` etc. Depending on which PR is going to be merged first we need to align constants used...